CMakeLists.txt

```diff
@@ -263,26 +263,14 @@ install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/tests/TestHarness DESTINATION ${CM
         PATTERN "__pycache__" EXCLUDE
         PATTERN "CMakeFiles" EXCLUDE)
 
-if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.21)
-   # Cmake versions < 3.21 did not support installing symbolic links to a directory via install(FILES ...)
-   # Cmake 3.21 fixed this (https://gitlab.kitware.com/cmake/cmake/-/commit/d71a7cc19d6e03f89581efdaa8d63db216184d40)
-   # If/when the lowest cmake version supported becomes >= 3.21, the else block as well as the postinit and prerm scripts
-   # would be a good option to remove in favor of using the facilities in cmake / cpack directly.
```

Comment:
I'm not sure what exactly this is suggesting we do, particularly since the install script does a runtime discovery of where to install the package. I don't think there is anything analogous in cpack. So idk.. I just removed the comment.. if someone has something they want back we can put something back.